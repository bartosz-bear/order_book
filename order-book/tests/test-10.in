{"type":"Iceberg","order":{"direction":"Sell","id":1,"price":14,"quantity":100,"peak":20}}
{"type":"Iceberg","order":{"direction":"Buy","id":2,"price":15,"quantity":90,"peak":25}}