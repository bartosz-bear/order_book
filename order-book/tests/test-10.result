{"buyOrders": [], "sellOrders": [{"id": 1, "price": 14, "quantity": 20}]}
{"buyOrders": [], "sellOrders": [{"id": 1, "price": 14, "quantity": 10}]}
