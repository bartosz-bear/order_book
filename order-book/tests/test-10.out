{"sellOrders": [], "buyOrders": [{"id": 1, "price": 100, "quantity": 50}]}
{"buyOrderId": 1, "sellOrderId": 1, "price": 100,  "quantity": 50}
{"buyOrderId": 1, "sellOrderId": 1, "price": 100,  "quantity": 50}
{"sellOrders": [{"id": 1, "price": 100, "quantity": 100}], "buyOrders": []}
